v2/src/modules/address/ModulePortfolioList.vue

```diff
@@ -0,0 +1,251 @@
+<template>
+    <div class="pa-4 pa-sm-6">
+        <module-add-adress-to-porfolio />
+        <module-add-adress-to-porfolio address="0xC5cA2Dd997E7Aa97f2DE490e329578Be27e9E644" />
```

Comment:
Is this still being used? Seems like it's being used when you click on the star button